actions/workflows/email.errored.server.users.yaml

```diff
@@ -0,0 +1,43 @@
+version: 1.0
+
+description: Executes the email.server.users action for errored VMs to informing users about them
+
+input:
+  - cloud_account
+  - project_id
+  - email_cc
+  - smtp_account
+  - test_override
+  - test_override_email
+
+vars:
+  - stdout: null
+  - stderr: null
+  - result: null
+
+output:
+  - result: <% ctx().result %>
+
+tasks:
+  send_emails:
+    action: stackstorm_openstack.email.server.users
+      cloud_account=<% ctx().cloud_account %>
+      project_identifier=<% ctx().project_id %>
+      email_cc=<% ctx().email_cc %>
+      smtp_account=<% ctx().smtp_account %>
+      test_override=<% ctx().test_override %>
+      test_override_email=<% ctx().test_override_email %>
+      query_preset=<% 'servers_errored' %>
+      properties_to_select=<% ['id', 'name', 'status', 'updated', 'user_name', 'user_email'] %>
+      header=<% '/opt/stackstorm/packs/stackstorm_openstack/email_templates/header.html' %>
+      message=<% 'The following VMs are in an errored state. Please submit a ticket if they are needed or delete them if they are not.' %>
+      footer=<% '/opt/stackstorm/packs/stackstorm_openstack/email_templates/footer.html' %>
+      subject=<% 'Errored VMs' %>
+      send_as_html=<% true %>
+      names=<% [] %>
+      name_snippets=<% [] %>
+      days=<% 1 %>
```

Comment:
This doesn't look quite right? I think it should just be
`days=1`

Also since it's not used, is it worth setting this to 0 to make it clear it's effectively None

Reply:
The <% %> doesn't make any difference here, just though it fit better with the other parameters which needed them.